c2cgeoportal/scaffolds/update/CONST_Makefile_tmpl

```diff
@@ -286,7 +286,6 @@ TEMPLATES_FILES = $(shell find $(PACKAGE)/templates -type f -print)
 
 # Templates
 TEMPLATE_EXCLUDE += .build print/templates \
-	templatecache.js.mako \
```

Comment:
Yes, if the file is named `templatecache.js.mako` but he is named `templatecache.mako.js` :-)